phpseclib/Crypt/Rijndael.php

```diff
@@ -190,29 +190,6 @@ class Rijndael extends Base
     var $kl;
 
     /**
-     * Default Constructor.
```

Comment:
well, it's not documenting anything - it just randomly stops, and another docblock started